### ISSUE TITLE
Black border around window

### ISSUE THREAD

**Issue by melgu** [diamondburned/dissent#296]
As you can see in the screenshot, I always have a large black border around the Dissent window. I already tried reinstalling the app.

![Dissent Black Border](https://github.com/user-attachments/assets/b73c5612-0c84-4286-996f-06fa7437519d)

**Comment by melgu** [comment]
By the way, this issue still persists with the nightly build I just tested in relation to the crashing bug (#297).

**Comment by tfuxu** [comment]
I was able to reproduce this, but for me the border was transparent. This might be a GTK/Adw Windows-specific bug, as Gaphor also has this, although the border is smaller.

**Comment by tfuxu** [comment]
Yep, this is a GTK issue: https://gitlab.gnome.org/GNOME/gtk/-/issues/7085 although this was reported on Linux, I guess the issue is same for Windows.
